test/game_board/building_tile_tests.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from stone_age.game_board.building_tile import BuildingTile
+from stone_age.game_board.simple_types import Player
+from stone_age.game_board.arbitrary_building import ArbitraryBuilding
+from stone_age.game_board.variable_building import VariableBuilding
+from  stone_age.game_board.simple_building import SimpleBuilding
+from stone_age.interfaces import InterfacePlayerBoardGameBoard
+from stone_age.simple_types import HasAction, ActionResult
+from stone_age.simple_types import PlayerOrder
+from stone_age.simple_types import Effect
+
+class TestBuildingTile(unittest.TestCase):
```

Comment:
I think these tests will not pass. We changed `InterfacePlayerBoardGameBoard` meanwhile, now it evaluates `assert False` in every method. You have to create mock of this class to be able to do unittests with it.

Reply:
I tried to mock that methods, but with that mocking mypy tests are not passed. And I have no idea why, I did that corresponding to documentation, but mypy points to "Function assingment" anyway and tests fail, despite the fact it is written in the documentation that Mock class is an exception and i need to assign Mock to a function. So I at least tried to pass mypy...

Reply:
Can we change somehow that Interface so that I don`t need to sacrifice code quality to take that to a working state?